docs/code/EntryPoints.md

```diff
@@ -0,0 +1,295 @@
+﻿# Entry Points And Helper Classes 
+
+## Overview
+
+An 'entry point', is a representation of a ML.NET type in JSON format. Entry points are used to serialize and deserialize an ML.NET type in JSON. 
+It is also the recommended way to interface with other languages. 
+Defined based on entry points, experiments are entry points DAGs, and respectively, entry points are experiment graph nodes.
```

Comment:
> Defined based on entry points, experiments are entry points DAGs, and respectively...

Could this be rephrased? I'm not quite sure what it is mean to express.

Experiments